wallaby.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (wallaby) {
```

Comment:
It's opt-in, it uses our existing test runner. You don't need it to run the tests, it just does it all the time. At work, we use NCrunch in VS, but that just works. Wallaby doesn't so well in our context, mainly because we have no working source maps.